src/ser.rs

```diff
@@ -90,30 +98,50 @@ impl<'py> Serialize for SerializePyObject<'py> {
             for (k, v) in simple_items {
                 let key = table_key(k)?;
                 let value = self.with_obj(v);
+                let v_ob_type = v.get_type_ptr() as usize;
+                if v_ob_type == lookup.none && self.omit_none {
+                    continue;
+                }
```

Comment:
this isn't required, `None` values are already excluded.

Reply:
This could be a nested item, say `{"x": None, "y": [1, 2, None]}`, right?
The exclusion above is excluding `"x"`, but not `None` in "y". Not sure if I understand it correctly.

Reply:
`y` is a list, not `None`.